job/job.go

```diff
@@ -74,16 +75,21 @@ func (*jobService) GetByName(ctx context.Context, name string) (*jobTypes.Job, e
 	return &job, err
 }
 
-func (*jobService) RemoveJobFromDb(jobName string) error {
```

Comment:
@morpheu could you rename this to RemoveJob ?

Reply:
Sure